dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/AccessStatus.kt

```diff
@@ -0,0 +1,11 @@
+package org.dataland.datalandcommunitymanager.model.dataRequest
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+/**
+ * An enum with the different access statuses of a data request
+ */
+@Schema(
+    enumAsRef = true,
+)
+enum class AccessStatus { Declined, Granted, Pending, Revoked, Public }
```

Comment:
i know this is nitpicky but perhaps change order of "Public" and "Revoked" to have it alphabetically sorted?

Reply:
Geändert